### DISCUSSION TITLE
Availability in Google Play

### DISCUSSION THREAD

**Discussion by arnowelzel** [rustdesk/rustdesk#7109] [Q&A] [answered]
I know, that the Android client for RustDesk is available at F-Droid (https://f-droid.org/de/packages/com.carriez.flutter_hbb/). However this requires installing the F-Droid app first and not every company policy allows installing apps from sources other than Google Play.

It would be helpful to have RustDesk available in Google Play as well.

**Comment by xlionjuan** [comment]
Too many bad guy use RustDesk to scam others, it is hard to solve

**Reply by arnowelzel** [reply]
And those bad guys are not able to use F-Droid or ask their victims to install an APK after enabling the use of other sources for apps?

**Reply by xlionjuan** [reply]
yes, because those being scam don't know how to use methods other then google play to install apps.

**Reply by arnowelzel** [reply]
Is there an FAQ entry for this? This might be helpful to avoid questions in the future and also questions about the integration in MDM solutions where only "offical" sources are allowed.

**Reply by MichaelUray** [reply]
Bad guys could use any other remote control software as well.
Why to remove the app for 99.9 % of the users just because 0.1% use it on a bad way?

**Reply by xlionjuan** [reply]
@MichaelUray other remote software like TV or Anydesk has their algorithm or something to detect bad behaviour

But RustDesk don't, and implement this is also not easy 🥲

**Reply by arnowelzel** [reply]
@MichaelUray I agree @xlionjuan - big companies have more resources to do a monitoring on their servers to identify and block suspicous activities. Maybe asking for a little fee for the app might help - people and companies who really need the app shouldn't have an issue to pay something, but bad guys would have a hard time convincing people to **purchase** an app just because someone asks for this.

**Reply by xlionjuan** [reply]
According to their Twitter, both platform has bad people using RustDesk for scam, both store has been taken down RustDesk before, but only AppStore released again, but not Play Store, I don't want to offensive those people that being scammed, but truth is, it is high possibility that those people may not have enough knowledge to not being scammed.

**Reply by xbyte99** [reply]
@xlionjuan - Don't you think the Scam Attention Alert and Countdown timer which was introduced in a recent version when starting Screen Capture in the RustDesk App is a sufficient countermeasure, even for very novice users who might be tricked into installing RustDesk?

These unavoidable countermeasures should be clearly communicated to the Google Play Store decision makers to clear RustDesk's "rap sheet".
And I really cannot stress this enough - in an enterprise environment, Android devices are usually centrally managed and restricted by policies, allowing app installations exclusively from the Google Play Store. 
By not getting clearance for the Google Play Store you're losing companies/users who are looking for a holistic remote support solution, which RustDesk definitely could be.

@arnowelzel - Having a store fee for the app is counterproductive in an enterprise environment. Paid apps are usually not accepted, as a company employee would need to buy the app individually in their Play Store. Apps with commercial goals on enterprise level offer subscriptions/accounts which are handled outside the Play Store with company contracts. (For example the Microsoft 365 app).
And also for a purely personal support situation (helping family and friends), a paid app is off-putting and it'll be hard to convince someone to buy (many don't even have a payment method associated with their accounts) the app when there are other app who offer the functionality for free.

**Reply by xlionjuan** [reply]
@xbyte99 
Dreams are beautiful, but reality is cruel. It need a solution to **prevent** this, not just said **We've WARM people before they using**, Google **WON'T** care about this, they only care people using their platform and got scammed, and dammm, maybe politics again, I think most people here know where RustDesk come from, just because this, they just ignore everything and start insult and DDoS(maybe) RustDesk.

**Reply by xbyte99** [reply]
@xlionjuan 
The RustDesk App was taken down in its 1.1.10-1 version state which was indeed very basic and lacked any clear scam warning message and countdown. Having it removed at that time was probably for the best, regarding possible abuse. 
But I am rather confident that with these warning adjustments and the fact that its source is completely transparent on GitHub, Google will now permit it on the Play Store. Definitely worth a try.

**Comment by rustdesk** [comment, marked as answer]
https://github.com/rustdesk/rustdesk/discussions/5660
